mini-pools-ios/ViewControllers/AddContributionViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  AddContributionViewController.swift
+//  mini-pools-ios
+//
+//  Created by Kaplún, Sebi on 12/20/17.
+//  Copyright © 2017 Kaplún, Sebi. All rights reserved.
+//
+
+import UIKit
+
+class AddContributionViewController: UIViewController {
+
+    var users : [PersonModel] = []
+    var poolId : Int?
+    weak var delegate : AddContributionDelegate?
+    let pickerView = UIPickerView()
+    
+    @IBOutlet weak var pickerTextField: UITextField!
+    @IBOutlet weak var amountField: UITextField!
+    @IBOutlet weak var noteField: UITextField!
+    @IBOutlet weak var backView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        pickerView.delegate = self
+        amountField.delegate = self
+
+        pickerTextField.inputView = pickerView
+        pickerTextField.text = users[0].name
+        
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.dismiss(_:)))
+        backView.addGestureRecognizer(tap)
+        
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(AddContributionViewController.keyboardWillShow), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(AddContributionViewController.keyboardWillHide), name: NSNotification.Name.UIKeyboardWillHide, object: nil)
+    
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    @objc func keyboardWillShow(notification: NSNotification) {
+            self.view.frame.origin.y = -50
```

Comment:
magic numbers. 
also, why not autolayout?

Reply:
@ShayDavidson  wanted to move this when the keyboard appears